src/dmd/dmodule.d

```diff
@@ -793,26 +793,26 @@ extern (C++) final class Module : Package
                         if (i >= eBuf.length)
                         {
                             error("surrogate UTF-16 high value %04x at end of file", u);
```

Comment:
I don't know how to trigger this.

Reply:
With a _really_ malformed file. Try using a hex editor and tack a surrogate UTF-16 high value to the end of it.

Reply:
Can you present an example? I tried but it triggers other errors. Maybe I missed something else. I don't think this is mandatory in this PR anyway, but would be cool to test this.